app/src/main/java/com/mithridat/nonoconverter/backend/solver/NonogramSolver.java

```diff
@@ -2,35 +2,25 @@
 
 import android.os.AsyncTask;
 
-import com.mithridat.nonoconverter.backend.Field;
+import com.mithridat.nonoconverter.backend.nonogram.Field;
+import com.mithridat.nonoconverter.backend.nonogram.Nonogram;
 
-import java.util.Arrays;
 import java.util.HashSet;
 
 /**
  * Nonogram solving class
  */
 public class NonogramSolver {
```

Comment:
Is it possible to make all fields `private` and create getters for all except `AsyncTask`?

Reply:
You can make getters `package-private` if you want.